shopgate/pwa
# Added an action that allows to disable/enable the login form

# Description

In this pull request we've added a new action that allows to lock/disable the login field.
This is especially useful for external login providers that need to fetch additional data.

## Type of change

Please add an "x" into the option that is relevant:

- [  ] Bug Fix :bug: (non-breaking change which fixes an issue)
- [ x ] New Feature :rocket: (non-breaking change which adds functionality)
- [  ] Breaking Change 💣 : (fix or feature that would cause existing functionality to not work as expected)
- [  ] Polish :nail_care: (Just some cleanups)
- [  ] Docs :memo: (Changes in the documentations)
- [  ] Internal :house: Only relates to internal processes.

## How to test it

- Install `@shopgate/auth-facebook` extension in the `PWA-1106` branch.
- Login using facebook.
- All login form buttons should be disabled.
